pytorch_lightning/trainer/trainer.py

```diff
@@ -1182,6 +1182,8 @@ def _call_teardown_hook(self, model: 'pl.LightningModule') -> None:
         self.teardown(stage=fn)
         model.teardown(stage=fn)
 
+        self.logger_connector.teardown()
+
```

Comment:
Maybe this should go into `_post_dispatch` with the rest of our internal teardowns

Reply:
Done !